### PR TITLE
basic outline for readme.md

### LINKED ISSUE #4: Add a proper README.md
Right now we have not enough documentation to help the contributors and users. A nice Documentation for README.md would be a great help for the project.

Please include following with proper formatting.

Brief intro to the repo (eg: Client Server Application to upload images to server/ S3 bucket.)
Getting Started (how to set up and run)
Dependencies
Contribution guidelines
Versioning
Authors

### PR DESCRIPTION
**Pull request by grlwholifts**
Created a basic outline for readme.md
Contributors can add/edit it accordingly. 
fixes #4 
cc: @tharakanandakumara @ShanikaEdiriweera

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
+
+# Welcome to Sign
+
+## Brief intro to the repo 
+This is a Client Server Application to upload images to server/ S3 bucket
```
Comment (ShanikaEdiriweera) [inline README.md:6]: This is an admin dashboard application for a school attendance marking system.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Getting Started
+Learn how to setup this project and run it on your local machine.
```
Comment (ShanikaEdiriweera) [inline README.md:9]: this is a simple php application. So you can add the steps how to clone the repo and run on the local apache server

File: README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+## Dependencies
+This project uses the following dependecies:
```
Comment (ShanikaEdiriweera) [inline README.md:12]: You could go through the code and find the dependancies. eg: bootstrap, chart library, excel handling library etc.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+This is a Client Server Application to upload images to server/ S3 bucket
+
+## Getting Started
+Learn how to setup this project and run it on your local machine.
+
+## Dependencies
+This project uses the following dependecies: 
+
+## Contribution guidelines
+Some basic guidelines for contribution are:
```
Comment (ShanikaEdiriweera) [inline README.md:15]: Add a general guideline to contribute

File: README.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+Learn how to setup this project and run it on your local machine.
+
+## Dependencies
+This project uses the following dependecies: 
+
+## Contribution guidelines
+Some basic guidelines for contribution are: 
+
+## Versioning
+The version system is as follows:
```
Comment (ShanikaEdiriweera) [inline README.md:18]: Add a general guideline

File: README.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+This project uses the following dependecies: 
+
+## Contribution guidelines
+Some basic guidelines for contribution are: 
+
+## Versioning
+The version system is as follows: 
+
+## Authors
+Awesome people who made this repo are:
```
Comment (ShanikaEdiriweera) [inline README.md:21]: Add me and tharaka

**Comment by ShanikaEdiriweera** [comment]
@ruchikabgosain I merged your PR  :1st_place_medal: 
Anyway I am not closing this issue #4  since there are more improvements needed to the file. I added review comments to the PR.

So you can complete them and do another PR. Please confirm whether you are doing it or not.

If @ruchikabgosain is not working on the improvements anyone can take this issue.

**Comment by grlwholifts** [comment]
Yup, I will do this. Thanks!

**Pull request merged**